Corona-Warn-App/src/main/java/de/rki/coronawarnapp/contactdiary/util/CWADateTimeFormatPatternFactory.kt

```diff
@@ -0,0 +1,32 @@
+package de.rki.coronawarnapp.contactdiary.util
+
+import org.joda.time.format.DateTimeFormat
+import java.util.Locale
+
+object CWADateTimeFormatPatternFactory {
+
+    fun Locale.shortDatePattern(): String {
+        DateTimeFormat.patternForStyle("S-", this).fixYear().also {
```

Comment:
Can't we just hardcode the patterns i.e. `"dd/MM/yyyy"` for each case? The current implementation seems very fragile.

Reply:
sure, I can change it. But then wen we would be forced to change code anytime we want to support a new region.